docs/features/web-api-template.md

```diff
@@ -24,21 +24,7 @@ When installed, the template can be created with shortname: `arcus-webapi`:
 
```

Comment:
Let's drop a note here on the version we have under `Web API Project Template` or so with a link maybe?

Reply:
You mean specifying which version this is?

Reply:
List all the versions of the documentation such as:

> - Latest
> - [v0.1](/v0.1/features/web-api-template)

Reply:
I did on the bottom a list of the older versions? Isn't that enough?

Reply:
LGTM, missed that sorry!